ui/src/pages/Deployment.vue

```diff
@@ -5,6 +5,7 @@
         :deployment="deployment"
         @update="deploymentSubscription.refresh"
         @delete="routeToDeployments"
+        :deployment-disable-toggle="true"
```

Comment:
```suggestion
        deployment-disable-toggle
```
The presence of this attr at all evaluates to `true`